docs/habitat.md

```diff
@@ -20,10 +24,10 @@ After creating a Chef Habitat package for an Chef InSpec profile (see CLI comman
 hab start adamleff/inspec-profile-frontend1
 ```
 
-The Chef Habitat Supervisor will install Chef InSpec and execute your profile in a loop. By default, the loop runs every 300 seconds but can be changed via the `sleep_time` configuration value:
+The Chef Habitat Supervisor will install Chef InSpec and execute your profile in a loop. The loop is controled by two variables the `interval` and the `splay`. The `interval` is a set time you want inspec to run the default is `1800` seconds. The `splay` is a random generated sleep time so that if you don't have a thundering herd scenario when sending your report to an exernal server like Chef Automate. The defualt for the `splay` is also 1800. You can also set the `splay_first_run` so that when your habitat package is started or updated it will wait a random period of time between 0 and the number set for the `splay_first_run` before running InSpec. The default of the `splay_first_run` is 0.
 
 ```bash
-HAB_INSPEC_PROFILE_FRONTEND1="sleep_time = 60" hab start adamleff/inspec-profile-frontend1
+HAB_INSPEC_PROFILE_FRONTEND1="interval = 60" hab start adamleff/inspec-profile-frontend1
```

Comment:
Consider using different example path - avoid former employees in examples

Reply:
Roger yep I'll switch that out for `effortless/audit-baseline`